plugins/@grouparoo/prometheus/.gitignore

```diff
@@ -0,0 +1,120 @@
+.DS_Store
```

Comment:
It's Grouparoo convention not to add a per-plugin `.gitignore` - we use the global one at the root of the monorepo.